.travis.yml

```diff
@@ -1,12 +1,21 @@
 language: node_js
+
 cache:
   directories:
     - 'node_modules'
+
 node_js:
   - '10'
+
 before_install:
   - cd react
+  - yarn add coveralls
```

Comment:
Why this isn't in the dev dependencies in the `package.json`?

Reply:
This dependency is only useful on CI process